crates/fuel-gas-price-algorithm/gas-price-analysis/src/charts.rs

```diff
@@ -11,44 +11,44 @@ pub fn draw_gas_prices(
     const EXEC_GAS_PRICE_COLOR: RGBColor = RED;
     const DA_GAS_PRICE_COLOR: RGBColor = BLUE;
     let min = 0;
-    let max = *gas_prices.iter().max().unwrap();
+    let max = *da_gas_prices.iter().max().unwrap();
 
     let mut chart = ChartBuilder::on(drawing_area)
         .caption(title, ("sans-serif", 50).into_font())
         .margin(5)
         .x_label_area_size(40)
-        .y_label_area_size(60)
-        .right_y_label_area_size(40)
+        .y_label_area_size(100)
+        .right_y_label_area_size(100)
         .build_cartesian_2d(0..gas_prices.len(), min..max)
         .unwrap();
 
     chart
         .configure_mesh()
-        .y_desc("Gas Price")
+        .y_desc("DA Gas Price")
         .x_desc("Block")
         .draw()
         .unwrap();
 
-    chart
-        .draw_series(LineSeries::new(
-            gas_prices.iter().enumerate().map(|(x, y)| (x, *y)),
-            GAS_PRICE_COLOR,
-        ))
-        .unwrap()
-        .label("Gas Price")
-        .legend(|(x, y)| PathElement::new(vec![(x, y), (x + 20, y)], GAS_PRICE_COLOR));
+    // chart
+    //     .draw_series(LineSeries::new(
+    //         gas_prices.iter().enumerate().map(|(x, y)| (x, *y)),
+    //         GAS_PRICE_COLOR,
+    //     ))
+    //     .unwrap()
+    //     .label("Gas Price")
+    //     .legend(|(x, y)| PathElement::new(vec![(x, y), (x + 20, y)], GAS_PRICE_COLOR));
 
     // Draw the exec gas prices
-    chart
-        .draw_series(LineSeries::new(
-            exec_gas_prices.iter().enumerate().map(|(x, y)| (x, *y)),
-            EXEC_GAS_PRICE_COLOR,
-        ))
-        .unwrap()
-        .label("Exec Gas Price")
-        .legend(|(x, y)| {
-            PathElement::new(vec![(x, y), (x + 20, y)], EXEC_GAS_PRICE_COLOR)
-        });
+    // chart
+    //     .draw_series(LineSeries::new(
+    //         exec_gas_prices.iter().enumerate().map(|(x, y)| (x, *y)),
+    //         EXEC_GAS_PRICE_COLOR,
+    //     ))
+    //     .unwrap()
+    //     .label("Exec Gas Price")
+    //     .legend(|(x, y)| {
+    //         PathElement::new(vec![(x, y), (x + 20, y)], EXEC_GAS_PRICE_COLOR)
+    //     });
```

Comment:
Can we remove this? Or is it useful to be able to toggle this? Then perhaps these sections could be broken out to separate functions which could be conditionally invoked based on some flag for example.

Reply:
It's noise right now since, but I think we should keep it for long-term.